src/geowombat/core/api.py

```diff
@@ -277,12 +277,12 @@ class open(object):
         mosaic (Optional[bool]): If ``filename`` is a ``list``, whether to mosaic the arrays instead of stacking.
         overlap (Optional[str]): The keyword that determines how to handle overlapping data if ``filenames``
             is a ``list``. Choices are ['min', 'max', 'mean'].
-        nodata (Optional[float | int]): A 'no data' value to set. Default is 0.
-            The 'no data' value is only used under these conditions:
-                1. if a ``geowombat.config.update`` context is used
-                2. if ``filename`` is a list or search string
-            Otherwise, if ``geowombat.open`` is used to open a single image without a config context then no
-            warping will be applied and, therefore, `nodata` will not apply.
+        nodata (Optional[float | int]): A 'no data' value to set. Default is ``None``.
+            If ``nodata`` is ``None``, the 'no data' value is set from the file metadata. If ``nodata`` is given,
+            then the file 'no data' value is overridden.
+
+            .. note:: The ``geowombat.config.update`` overrides this argument. Thus, preference is always given
+                to ``geowombat.config.update`` not ``None`` > ``nodata`` not ``None`` > file 'no data'.

```

Comment:
You lost me on the 
`not ``None`` > ``nodata`` not ``None`` > file 'no data'.`
section

Reply:
Okay, I modified the docstring.